### ISSUE TITLE
Error creating invoice

### ISSUE THREAD

**Issue by dbrisser** [kimai/kimai#1011]

**Comment by dbrisser** [comment]
Hi, I started using Kimai this month and tried to create an invoice for this month. I have all the fields filled out and when I click the create button I a new page opens with the error "Missing time period" . I have tried adjusting the time period to be d/m/y or m/d/y as well as other variations of the date however the error persists. Is there something I'm missing? I have all my dates setup in preferences as dd/mm/yy.

**Comment by kevinpapst** [comment]
Can you sent us all the relevant settings and a screenshot of your invoice screen just before submitting. 
Are there javascript errors in your console? 
Is there something in the Kimai log (check the debug panel)?

**Comment by simonschaufi** [comment]
No response

**Comment by Lagneaux** [comment]
Kimai : v1.3.1.1394 and installed on an external and shared server

Hi,

While all fields in the "Invoice" tab are completed, when you click "Create", an error message is displayed : «Missing time period».

I guess the error comes from the print.php and private_func.php scripts since the message url come from: .../extensions/ki_invoice/print.php

What i have to do?

Thank you in advance for your help ...

Regards

**Comment by simonschaufi** [comment]
We don't fully support english date formats yet but we have something to review to make it work. See https://github.com/kimai/kimai/pull/1121

**Comment by Lagneaux** [comment]
Hi,

I still have my **invoice edition** problems. In my previous post, I posted my date settings.

I tried several possible combinations with / or . and also reverse the order in the dates parameters between d, m, y but nothing helps. I still have the same error message:
"**Missing time period**".

What would be the code change to do? And in which file(s)? In which lines?

I tried to find the post that would help me, but without success!

Thank you in advance for always helpful help.
Yours truly
